### PR TITLE
dighal task dialogue

### PR DESCRIPTION
**Pull request by allister-translator**
that should be all of it for dighal i think

### CODE REVIEW

File: Translated/Dialog/Dighal/ssq_012222.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name_7417#0,"""Dighal"""
+ssq_012222_0010#0,"""Alright, looks like you didn't have any trouble being able to gather them."""
```
Comment (DabirSA) [inline Translated/Dialog/Dighal/ssq_012222.csv]: "didn't have any trouble Gathering them."?

Comment (allister-translator) [reply Translated/Dialog/Dighal/ssq_012222.csv]: yeah that was bugging me too, sec

Comment (allister-translator) [reply Translated/Dialog/Dighal/ssq_012222.csv]: fixt

File: Translated/Dialog/Dighal/ssn_012220.csv [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+ssn_012220_0002#0,"""Outside the city, there's a wealth of vegetables, fruits, Ores and Tames meat to be harvested."""
+ssn_012220_0003#0,"""Those fresh ingredients and strong materials are sure to benefit you in your battles to come."""
+ssn_012220_0101#0,"""Yo, how's it going?"""
+ssn_012220_0102#0,"""Now that you've been getting some new weapons and upgrading them a bit, you might notice they'll require some new types of Ore."""
+ssn_012220_0103#0,"""You want to learn more about those Ores?"""
+ssn_012220_0201#0,"""Dualomite is a purple Ore with a jagged formation. It's an Ore similar in colour to Photon Chunks, but with different properties."""
+ssn_012220_0202#0,"""They're both gorgeous gems, but they can also be used at the Item Lab as catalysts for strengthening your equipment."""
+ssn_012220_0203#0,"""Many can be found dotting the walls of the Southern Aerio caverns. It'd do you good to go and try finding some."""
+ssn_012220_0301#0,"""Mm, I see... That's unfortunate."""
+ssn_012220_0302#0,"""Well, whenever you'd like to learn about Ores, let me know at any time."""
```
Comment (DabirSA) [inline Translated/Dialog/Dighal/ssn_012220.csv]: "whenever... at any time" sentence construction is a bit awkward.

Comment (allister-translator) [reply Translated/Dialog/Dighal/ssn_012220.csv]: grammarception fixed

**Pull request merged**